lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml

```diff
@@ -122,54 +143,77 @@ prefix_list_out:
 
 route_map_in:
   kind: string
-  config_get_token_append: '/^route-map (\S+) in$/'
-  config_set_append: '<state> route-map <str> in'
   default_value: ''
+  cli_nexus:
+    config_get_token_append: '/^route-map (\S+) in$/'
+    config_set_append: '<state> route-map <str> in'
+  cli_ios_xr:
+    config_get_token_append: '/^route-policy (\S+) in$/'
+    config_set_append: ' <state> route-policy <str> in'
 
 route_map_out:
   kind: string
-  config_get_token_append: '/^route-map (\S+) out$/'
-  config_set_append: '<state> route-map <str> out'
   default_value: ''
+  cli_nexus:
+    config_get_token_append: '/^route-map (\S+) out$/'
+    config_set_append: '<state> route-map <str> out'
+  cli_ios_xr:
+    config_get_token_append: '/^route-policy (\S+) out$/'
+    config_set_append: ' <state> route-policy <str> out'
 
 route_reflector_client:
   kind: boolean
   config_get_token_append: '/^route-reflector-client$/'
-  config_set_append: '<state> route-reflector-client'
+  config_set_append: ' <state> route-reflector-client'
   default_value: false
 
 send_community:
   auto_default: false
-  config_get_token_append: '/^send-community(?: .*)?/'
-  config_set_append: '<state> send-community <attr>'
-  default_value: 'none'
+  cli_nexus:
+    config_get_token_append: '/^send-community(?: .*)?/'
+    config_set_append: '<state> send-community <attr>'
+    default_value: 'none'
+  cli_ios_xr:
+    multiple:
+    # XR four seperate commands: send-community-ebgp  send-community-gshut-ebgp
+    # send-extended-community-ebgp  send-multicast-attributes
```

Comment:
Remove reference to `send-multicast-attributes` unless you're *sure* it's relevant to send-community.